aten/src/ATen/native/native_functions.yaml

```diff
@@ -254,17 +254,26 @@
 - func: chunk(Tensor self, int64_t chunks, int64_t dim=0) -> TensorList
 
 - func: clamp(Tensor self, Scalar min, Scalar max) -> Tensor
+  python_default_init:
+    min: NAN
```

Comment:
NAN defined here (or any other non-empty input like 2) will generate a default 'None' here, see [generation logic](https://github.com/pytorch/pytorch/blob/master/tools/autograd/gen_python_functions.py#L729) here. It should not be a correct behavior for other default args when generating the functions. The generation logic need to be fix, tracked the issue [here](https://github.com/pytorch/pytorch/issues/9941)